Rdkafka.xs

```diff
@@ -424,6 +424,34 @@ krdm_timestamp(msg,...)
     OUTPUT:
 	RETVAL
 
+HV*
+krdm_headers(msg)
+        rd_kafka_message_t* msg
+    PREINIT:
+        rd_kafka_resp_err_t err;
+        rd_kafka_headers_t *hdrs;
+        size_t len, i;
+        const char *name;
+        const void *value;
+    CODE:
+        RETVAL = newHV();
+        sv_2mortal((SV*)RETVAL);
+
+        err = rd_kafka_message_headers(msg, &hdrs);
+        if (err == RD_KAFKA_RESP_ERR_NO_ERROR) {
+            for (i = 0;; ++i) {
+                err = rd_kafka_header_get_all(hdrs, i, &name, &value, &len);
+                if (err == RD_KAFKA_RESP_ERR__NOENT)
+                    break;
+                hv_store(RETVAL, name, strlen(name), newSVpvn(value, len), 0);
```

Comment:
Can `hv_store` fail?

Reply:
yes it can

> These each store SV val with the specified key in hash hv, returning NULL if the operation failed or if the value did not need to be actually stored within the hash (as in the case of tied hashes). Otherwise it can be dereferenced to get the original SV*.
https://perldoc.perl.org/perlapi#hv_store